matrix-generator.js

```diff
@@ -1,5 +1,28 @@
 function matrixGenerator(rows, columns) {
-  // do work here
+  const matrix = [];
+
+  for (let i = 0; i < rows; i++) {
+    const newRow = [];
+
+    for (let j = 0; j < columns; j++) {
+      const randomNumber = Math.round((Math.random() * 100));
+
+      newRow.push(randomNumber);
+      newRow.sort();  // sorts by bitcode, buggy with numbers like 4 and 40, 5 and 50, etc
```

Comment:
Try and look up how to use the sort function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort